cmd/cloudtrail/write-events.go

```diff
@@ -163,16 +117,23 @@ func GetEvents(since time.Time, client *cloudtrail.Client, pages int) ([]*cloudt
 		},
 	}
 
-	maxPages := pages
+	pages := 0
+	paginator := cloudtrail.LookupEventsPaginator{}
+	hasPages := paginator.HasMorePages()
+
+	for pageIter := 10; hasPages; pageIter++ {
+		pages += pageIter
+	}
```

Comment:
This seems like an odd way of paginating? Can you explain what approach we're using here?